doc/rst/usingchapel/launcher.rst

```diff
@@ -341,3 +342,6 @@ Launcher Name  Description
 =============  ==========================================================
 mpirun         launch using mpirun (no mpi comm currently) 
 =============  ==========================================================
+
+.. |-| unicode:: U+2011 .. non-breaking hyphen
+  :trim:
```

Comment:
This seems a little unfortunate in that it makes the raw text harder to read or copy+paste, but I agree it makes the rendered table a little nicer.

Reply:
I didn't think about the copy+paste issue. Do you think it's worth doing?

Reply:
Probably the rendered text is the most important thing, so I learn towards keeping it.